drivers/ieee802154/Kconfig

```diff
@@ -37,7 +36,6 @@ config SYS_LOG_IEEE802154_DRIVER_LEVEL
 
 config IEEE802154_RAW_MODE
 	bool "Use IEEE 802.15.4 driver without the MAC stack"
-	default n if NET_L2_IEEE802154
```

Comment:
@tbursztyka Removing this is a no-op, but I'm not sure what this was meant to do. Just documentation?